packages/app-check/src/api.ts

```diff
@@ -97,6 +98,12 @@ export function initializeAppCheck(
 
   const appCheck = provider.initialize({ options });
   _activate(app, options.provider, options.isTokenAutoRefreshEnabled);
+  // Adding a listener will start the refresher and fetch a token if needed.
+  // This gets a token ready and prevents a delay when an internal library
+  // requests the token.
+  // Listener function does not need to do anything, its base functionality
+  // of calling getToken() already fetches token and writes it to memory/storage.
+  addTokenListener(appCheck, ListenerType.INTERNAL, () => {});
```

Comment:
Does it mean the refresher will never stop because there is always a listener? or will there be a case where the refresher is stopped while there is a listener?

Reply:
I don't know if I understand the question, when should the refresher be stopped? The refresher will always run at the appointed time but the REST call may be blocked by some of the new changes.

However this reminds me I missed something really important, which is that this shouldn't start if `isTokenRefreshEnabled` is false, and if the user later sets it to false, the listener should be removed.

Reply:
I don't think you need to remove this dummy listener in case `isTokenRefreshEnabled` is later set to false. It doesn't make any differences functionally, or does it?

Reply:
Ah, that's right,`setTokenAutoRefreshEnabled(appCheck, false)` already stops the refresher.